cpplib/src/Python/PythonInterface.cpp

```
@@ -849,6 +854,79 @@ extern "C" {
+		auto ce = diag.extractCells();
+
+		cpplib::voronoi::VoronoiFused vf(ce, cell.fracToCart());
+		vf.polyhedra.resize(all.points.size());
+
+		auto critical_points = cpplib::BaderOperator::GenerateInitialCriticalPoints(vf);
+
+		// Expand Point Net
+		// 1. Calculate theoretical radius.
+		
+		const auto ED_cutoff = cpplib::BaderOperator::GetOptimalRadius(3.0, 1.0E-6, ElectronDensitySplines, buildresult.atoms.types);
+		const size_t max_type = cpplib::ElectronDensitySplines.size();
```

Cppcheck (2.21.0)

[style] The function 'getAngleRad' is never used.

(unusedFunction)